base/client.jl

```diff
@@ -246,7 +246,7 @@ function exec_options(opts)
             # If we're doing a bug report, don't load anything else. We will
             # spawn a child in which to execute these options.
             let InteractiveUtils = load_InteractiveUtils()
-                InteractiveUtils.report_bug(arg)
+                invokelatest(InteractiveUtils.report_bug, arg)
```

Comment:
`@invokelatest` is a bit cleaner

Reply:
I tried to copy the style that was used in the `Distributed` case just below. I don't feel like this makes a huge difference in terms of readability in the single-arg case